packages/build/src/plugins_core/build_command.js

```diff
@@ -1,3 +1,4 @@
+import fsPromises from 'node:fs/promises'
```

Comment:
```suggestion
import { rm } from 'node:fs/promises'
```
[dust]: always prefer named import over default imports